cylc/flow/xtriggers/workflow_state.py

```diff
@@ -54,6 +54,7 @@ def workflow_state(
             The task status required for this xtrigger to be satisfied.
         message:
             The custom task output required for this xtrigger to be satisfied.
+
```

Comment:
That newline is functional, if you remove it, then `.. note::` isn't a directive:

![Screenshot from 2024-04-24 10-57-57](https://github.com/cylc/cylc-flow/assets/16705946/55ea5cf1-5592-4f05-aa7d-b87632e4a573)